railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -852,6 +852,28 @@ class TwoMigration < ActiveRecord::Migration::Current
         end
       end
 
+      test "db:prepare runs seeds once" do
+        require "#{app_path}/config/environment"
+        Dir.chdir(app_path) do
+          use_postgresql(multi_db: true)
+
+          rails "db:drop"
+          generate_models_for_animals
+          rails "generate", "model", "recipe", "title:string"
```

Comment:
I'm curious about this piece, it doesn't seem like we use `Recipe` in the test

Reply:
We need migrations for both databases, so when we call `db:prepare`, it creates both databases before running migrations. And we expect seeds to run only once. 
Otherwise only `animals` database will be created and seeds will run once without code changes.